docs/Users_Guide/reformat_stat_data.rst

```diff
@@ -46,160 +53,634 @@ Plots that are generated from the command line require a YAML configuration file
-plot and line type.
+The reformatted data contains new columns, based on the type of
+plot and line type.  Formats fall into the following categories:
+
+    .. dropdown::  by statistic name, statistic value and confidence values
+
+       - Needed by the following plot types in METplotpy:
+
+         - line
+         - bar
+         - box
+         - contour
+         - performance diagram
+         - revision box
+         - revision series
+         - Taylor diagram
+
+       - Unlabelled columns in the original .stat file are placed under statistic name, statistic value,
+         and confidence level columns (in addition to the common stat columns):
+
+         - stat_name
+         - stat_value
+         - stat_ncl
+
+            - lower level normal confidence limit
+
+         - stat_ncu
+
+            - upper level normal confidence limit
+
+         - stat_bcl
+
+            - lower level bootstrap confidence limit
+
+         - stat_bcu
+
+            - upper level bootstrap confidence limit
+
+       - The original format is converted from wide format (where each row corresponds to a unique
+         set of common stat column values) to longer format (where there are multiple rows
+         corresponding to the same common stat column values).
+
+           - The common stat columns are common to all line types for point-stat, grid-stat, and ensemble-stat (model, vx mask, description, fcst lead, etc.)
+             as described in the MET User's Guide:
+
+              - table 11.1 (point-stat)
+              - table 12.1 (grid-stat)
+              - table 13.1 (ensemble-stat)
+
+
+
+         .. dropdown:: Unformatted Example:
+
+            .. literalinclude:: ./figure/grid_stat_FV3_core_lsm1_020000L_20190521_020000V.stat
+
+         - In the unformatted example:
+
+           - each row represents unique data (tidy data)
+           - the number of columns is different for each row, due to each row representing a different linetype
+             (e.g. DMAP, NBRTC)
+           - numerous columns are unlabelled
+
+
+         .. dropdown:: Reformatted Example (truncated):
+
+            ..  literalinclude:: ./figure/reformatted_dmap_for_lineplot_output.txt
 
-**Reformatting for
-line, bar, box, contour, performance diagram, revision box, revision series, and taylor diagram**
 
-After reformatting, the .stat data contains the following columns (replacing the unlabelled
-columns in the original .stat file and converting from wide to long format):
+         -  In the reformatted example:
 
-Two columns for statistics
+            - reformatted data file contains only one linetype (in this example DMAP), as requested in the configuration file
 
-- stat_name
+            - multiple rows have duplicate common stat column values, resulting in more rows
 
-- stat_value
+              - note that for the rows containing the first **fy** and **oy** stat_names, the common stat columns of
```

Comment:
```suggestion
              - note that for the rows containing the first **FY** and **OY** stat_names, the common stat columns of
```
Changing these to upper case to be consistent with the MET documentation (https://met.readthedocs.io/en/latest/Users_Guide/grid-stat.html#id9)